watertap/unit_models/ion_exchange_0D.py

```diff
@@ -35,22 +39,60 @@
 from idaes.core.util.constants import Constants
 from idaes.core.util.config import is_physical_parameter_block
 from idaes.core.util.misc import StrEnum
+from idaes.core.util.exceptions import InitializationError, ConfigurationError
 
-from idaes.core.util.exceptions import InitializationError
 import idaes.core.util.scaling as iscale
 import idaes.logger as idaeslog
 
-from watertap.core import InitializationMixin
+from watertap.core import ControlVolume0DBlock, InitializationMixin
 
 __author__ = "Kurban Sitterley"
 
+
+"""
+REFERENCES
+
+LeVan, M. D., Carta, G., & Yon, C. M. (2019).
+Section 16: Adsorption and Ion Exchange.
+Perry's Chemical Engineers' Handbook, 9th Edition.
+
+Crittenden, J. C., Trussell, R. R., Hand, D. W., Howe, K. J., & Tchobanoglous, G. (2012).
+Chapter 16: Ion Exchange.
+MWH's Water Treatment (pp. 1263-1334): John Wiley & Sons, Inc.
+
+DOWEX Ion Exchange Resins Water Conditioning Manual
+https://www.lenntech.com/Data-sheets/Dowex-Ion-Exchange-Resins-Water-Conditioning-Manual-L.pdf
+
+Inamuddin, & Luqman, M. (2012).
+Ion Exchange Technology I: Theory and Materials.
+
+Vassilis J. Inglezakis and Stavros G. Poulopoulos
+Adsorption, Ion Exchange and Catalysis: Design of Operations and Environmental Applications (2006).
+doi.org/10.1016/B978-0-444-52783-7.X5000-9
+
+Michaud, C.F. (2013)
+Hydrodynamic Design, Part 8: Flow Through Ion Exchange Beds
+Water Conditioning & Purification Magazine (WC&P)
+https://wcponline.com/2013/08/06/hydrodynamic-design-part-8-flow-ion-exchange-beds/
+
+Clark, R. M. (1987). 
```

Comment:
You might've done this, but did you include the Clark ref in the rst file?

Reply:
Yes, Clark (1987) is in the rst file.